zlp-scheduler/app/controllers/admin_controller.rb

```diff
@@ -156,8 +156,8 @@ def view_result
     @cohort = Cohort.find(params[:cohort_id])
     # Scheduler_2.Generate_time_slots(@cohort)
     @date_dict = { "M" => "Monday", "T" => "Tuesday", "W" => "Wednesday", "TR" => "Thursday", "F" => "Friday"}
-    @results = TimeSlot.where(:was_conflict => false).order(cost: :asc,id: :asc).limit(10)
-    @conflicts = TimeSlot.where(:was_conflict => true).order(cost: :asc,id: :asc).limit(10)
+    @results = TimeSlot.where(:was_conflict => false).order(cost: :asc,id: :asc).limit(200)
+    @conflicts = TimeSlot.where(:was_conflict => true).order(cost: :asc,id: :asc).limit(100)
```

Comment:
I just realized that this is not filtering based on `cohort_id`. Could we add that so multiple cohorts can be supported?

Reply:
Good Catch! it should be fixed now